### ISSUE TITLE
aws_sqs_queue not restoring visibility_timeout_seconds to default value after removal of attribute

### ISSUE THREAD

**Issue by jochem** [hashicorp/terraform#7505]
### Terraform Version
Terraform v0.6.16

### Affected Resource(s)
- aws_sqs_queue

### Terraform Configuration Files
```hcl
resource "aws_sqs_queue" "backend_queue" {
    name = "backend"
    visibility_timeout_seconds = 90
    policy = "{*snip*}"
}
```

### Expected Behavior
According to https://www.terraform.io/docs/providers/aws/r/sqs_queue.html, the default value for the visibility_timeout_seconds attribute is 30. Upon the removal of this attribute from the resource the value should be set to 30.

### Actual Behavior
Upon removal, the value of the attribute stayed on the previously set value, 90 in this case.

### Steps to Reproduce
1. Create an aws_sqs_queue resource without specifying visibility_timeout_seconds
2. Run `terraform apply` to create the resource
3. Add the visibility_timeout_seconds attribute and set it to a value other than 30
4. Run `terraform apply` to modify the resource
5. Remove the visibility_timeout_seconds attribute
6. Run `terraform apply` to see no modifications are done

**Comment by stack72** [comment]
Hi @jochem 

Thanks for reporting the issue, sorry this has caught you out! You have found that the documentation (and the AWS API) defaults to 30, but yet in Terraform, we do not set a default

I will try and get this fixed today for the 0.7 release

Thanks

Paul

**Comment by catsby** [comment]
I opened #7818 to restore defaults for SQS Queues, thanks!
